apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -89,10 +102,30 @@ export function InstructorAIGenerateDrafts({
                   )}
                 </tbody>
               </table>
+
+              ${DestroyQuestionsModal(resLocals)}
             </div>
           </div>
         </main>
       </body>
     </html>
   `.toString();
 }
+
+function DestroyQuestionsModal(resLocals) {
+  return Modal({
+    id: 'destroyModal',
```

Comment:
`destroyQuestionsModal`? Why use the verb "destroy" when the verb used elsewhere is "delete"?

Reply:
done in c0a3160.